src/datasets/model.rs

```diff
@@ -276,21 +276,32 @@ impl Query {
     }
 }
 
+// We need to pass a reference for serde compatibility.
+#[allow(clippy::trivially_copy_pass_by_ref)]
+fn is_false(b: &bool) -> bool {
+    !*b
+}
+
 // QueryParams is the part of `QueryOptions` that is added to the request url.
 #[derive(Serialize, Debug, Default)]
 pub(crate) struct QueryParams {
-    #[serde(rename = "nocache")]
+    #[serde(rename = "nocache", skip_serializing_if = "is_false")]
     pub no_cache: bool,
```

Comment:
no_cache can be omitted too → optional

Reply:
from what I understand it can either be absent or true, `Option<bool>` create three states instead of two `None`, `Some(true)` and `Some(false)` the omission is handled by `skip_serializing_if`. Unless I misunderstand how `nocache` works and absent and `nocache=false` are different if so we need to trickle that statespace up to the interfaces too